packages/blade/src/components/TopNav/__tests__/TabNav.test.stories.tsx

```diff
@@ -23,35 +110,87 @@ TestBasicTabNav.play = async ({ canvasElement }) => {
 };
 
 export const TestOverflow: StoryFn<typeof TabNav> = (): React.ReactElement => {
+  return <TabNavExample />;
+};
+
+TestOverflow.play = async ({ canvasElement }) => {
+  const { getByRole, queryByRole } = within(document.body);
+  canvasElement.style.width = '100%';
+
+  await sleep(500);
+  const homeTab = getByRole('link', { name: 'Home' });
+  const payrollTab = getByRole('link', { name: 'Payroll' });
+  const paymentsTab = getByRole('link', { name: 'Payments' });
+  await expect(homeTab).toBeVisible();
+  await expect(payrollTab).toBeVisible();
+  await expect(paymentsTab).toBeVisible();
+
+  await sleep(500);
+
+  // reduce the width of the canvas to make the tabs overflow
+  canvasElement.style.width = '600px';
+  await sleep(500);
+
+  const moreTab = getByRole('button', { name: 'More' });
+  await userEvent.hover(moreTab);
+  await sleep(500);
+  await expect(queryByRole('menu', { name: 'More' })).toBeVisible();
+  await expect(queryByRole('menuitem', { name: 'Rize' })).toBeVisible();
+  await expect(queryByRole('menuitem', { name: 'Magic Checkout' })).toBeVisible();
+
+  canvasElement.style.width = '300px';
+  await sleep(500);
+  await expect(queryByRole('menuitem', { name: 'Payroll' })).toBeVisible();
+  await expect(queryByRole('menuitem', { name: 'Payments' })).toBeVisible();
+
+  canvasElement.style.width = '100%';
```

Comment:
what does width 100% check here?

Reply:
It doesn't check anything i'm resetting the canvas to be full width.